app/services/idv/step.rb

```diff
@@ -1,44 +1,27 @@
 # abstract base class for Idv Steps
 module Idv
   class Step
-    include VendorValidated
-
-    def initialize(analytics: nil, idv_form:, idv_session:, params:)
+    def initialize(idv_form:, idv_session:, params:)
       @idv_form = idv_form
       @idv_session = idv_session
-      @analytics = analytics
       @params = params
     end
 
-    def complete
-      form_valid? && vendor_valid?
-      track_event
-      complete?
-    end
-
     def form_valid?
       form_validate(params)
     end
 
-    def vendor_valid?
-      vendor_validate
-    end
-
-    def complete?
-      raise NotImplementedError "Must implement complete? method for #{self}"
-    end
-
     private
 
-    attr_accessor :analytics, :idv_form, :idv_session, :params, :form_result
+    attr_accessor :idv_form, :idv_session, :params, :form_result
```

Comment:
Sure, but this is not new to this PR. I only removed the first item. I was going to update this in a follow up PR to keep this one small. We don't need all of these to be attr_accessors. Do you want me to make this change here or later?